e2e/pages/events/removeOrderEvent.page.js

```diff
@@ -0,0 +1,13 @@
+const {I} = inject();
+
+module.exports = {
+  fields: {
+    reason: '#removeOrderReason',
+  },
+
+
+
```

Comment:
couple of spaces we could remove maybe :)

Reply:
This is an outdated version